participants/Hipool cloud disk.md

```diff
@@ -0,0 +1,89 @@
+# <Project Name>
+
+Hipool cloud disk
+
+
+
+## (0) Project Name and Team Slack Handles
+
+Hipool cloud disk
+
+Hangzhou Net Storage Technology Co., Ltd.
+
+
+
+## (1) Brief Project Description
+
+Provide network file storage services for market users and small and medium-sized enterprises.
+
+
+
+## (2) Link to Project UI
+
+Project link address (you can view it after registration and login)：[__http://hipool.io/__]
+
+
+
+## (3) What does your application/UI do?
+
+It is convenient for users to upload and download storage files.
+
+
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+video
+
+
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+All data comes from users, we do not own the data, and we also control sensitive data or illegal information.
+
+
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+There are 86T data
+
+
+
+## (7) How are you structuring the data?
+
+Video data
+
+
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+The plan is that when users save and upload any video files, they will be saved on the network.
+
+
+
+## (9)  What tech stack will you use to store the data?
+
+Currently lotus technology is temporarily used to store data.
```

Comment:
How are you planning for hot retrieval?  Powergate or an IPFS node may be good options here.